### PR TITLE
updated embedding input dim

### PR DESCRIPTION
**Pull request by hojinYang**
I updated the input dimension of the embedding matrix in the examples in the docs. 

1. For timestamp embedding, I updated tf.keras.layers.Embedding(len(timestamp_buckets) + 2, 32) to tf.keras.layers.Embedding(len(timestamp_buckets) + 1, 32). The index ranges from 0 to the size of the timestamp_buckets. That is, if the number of buckets is 1,000, we only need 1000 + 1 indexes, not 1000 + 2.

2.  For user embedding, I updatedtf.keras.layers.Embedding(len(unique_user_ids) + 2, embedding_dimension) to tf.keras.layers.Embedding(len(unique_user_ids) + 1, embedding_dimension), as mask embedding is never used in two-tower model. I set mask_token argument of StringLookup to 'None' to ignore mask embedding.

Although this update does not affect the execution of the tutorial, This will help readers understand how embedding works more clearly

### CODE REVIEW

**Comment by hojinYang** [comment]
> @googlebot

I signed it

File: docs/examples/basic_retrieval.ipynb [hunk 1]
```diff
@@ -406,9 +406,9 @@
       "source": [
         "user_model = tf.keras.Sequential([\n",
         "  tf.keras.layers.experimental.preprocessing.StringLookup(\n",
-        "      vocabulary=unique_user_ids),\n",
-        "  # We add 2 to account for unknown and mask tokens.\n",
-        "  tf.keras.layers.Embedding(len(unique_user_ids) + 2, embedding_dimension)\n",
+        "      vocabulary=unique_user_ids, mask_token=None),\n",
+        "  # We add additional embedding to account for unknown tokens.\n",
```
Comment (maciejkula) [inline docs/examples/basic_retrieval.ipynb:410]: Slight wording correction: "We add _an_ additional embedding..."

**Review by maciejkula — CHANGES_REQUESTED** [review]
Thanks, I left one minor comment.

File: docs/examples/basic_retrieval.ipynb (hunk 1 above)
Comment (hojinYang) [reply docs/examples/basic_retrieval.ipynb]: Sure! Just updated it. Many thanks!

**Review by maciejkula — APPROVED** [review]
